### PR TITLE
add namepattern example with some comments

### PR DESCRIPTION
**Pull request by hooger**
namePatterns are missing from the sample config. I added default values with some comments

### CODE REVIEW

File: sample.config.yaml [hunk 1]
```diff
@@ -90,3 +90,28 @@ logging:
...
+  user: :name
+
+  # Name of bridged Slack channels
+  #
+  # Available variables:
+  #
+  # name: name of the channel
+  # team: name of the team
+  # type: type of the room
+  room: :name
```
Comment (auscompgeek) [inline sample.config.yaml:110]: This isn't the default.

Comment (hooger) [reply sample.config.yaml:110]: I see. Can you tell me what is the default so I can update it accordingly?

Comment (Sorunome) [reply sample.config.yaml:110]: Thesea re the defaults https://github.com/Sorunome/mx-puppet-slack/blob/master/src/index.ts#L61-L64

Comment (hooger) [reply sample.config.yaml]: Thank you very much! I changed it according to the code @Sorunome linked. However I don't understand what it does. :( Would any of you be so kind the enlighten me?

Comment (Sorunome) [reply sample.config.yaml]: `[if?then,else]` are if-conditions in the name patterns

**Pull request merged**